pics/PICS_TRANSLATION.md

```diff
@@ -62,7 +68,9 @@ Suppression
 
 #### Traduction (français) 
 
+Atténuation
 
+Endiguement
 
 ### plan.png
```

Comment:
Pas évident le plan.png, voici une proposition: 


Objectif : R<1.  (+ le point pas en gras, IMHO ça prend de la place)
En commençant par un...

Confinement
Afin de repartir de zéro et (de) gagner du temps pour préparer la phase de...

Test, suivi, isolement
Si cela ne suffit pas pour atteindre R<1, en complétant avec "masque pour tous" et un confinement "disjoncteur" jusqu'à ce qu'arrive le...

Vaccin
si l'immunité ne dure pas, en vaccinant régulièrement. Si aucun vaccin n'est trouvé, en changeant nos habitudes jusqu'à ce que nous puissions traiter ou guérir facilement !

On  a choisi test, suivi,  isolement dans les traduction précédentes. (discutables mais gardons la cohérence pour le moment)

Reply:
L'originale utilise la forme "we" plusieurs fois. Pourquoi ne pas utiliser "nous" en français ?
Voici ma proposition: 

Objectif : R<1.
Commenç**ons** par un...

Confinement
Afin de repartir de zéro et (de) gagner du temps pour préparer la phase de...

**Tester, Suivre, Isoler**
Si cela ne suffit pas pour atteindre R<1, complét**ons** avec "masque pour tous" et un confinement "disjoncteur" jusqu'à ce qu'arrive le...

Vaccin
Si l'immunité ne dure pas, vaccin**ons** régulièrement. Si aucun vaccin n'est trouvé, change**ons** nos habitudes jusqu'à ce que nous puissions traiter ou guérir facilement !


Concernant "Test, suivi, isolement", je proposerais plutôt "Tester, Suivre, Isoler". L'infinitif transmet pour moi un message plus **actif** (cela ne va pas se faire tout seul...)

Reply:
J'aime bien le passage au «nous».
Pour l'instant j'ai conservé «Test, suivi, isolement» pour l'uniformité.
Ailleurs il y a des discussions pour remplacer «suivi» par «traçage», à suivre.

Reply:
oui

Reply:
pour test,suivi,isolement on verra plus tard, c'est pas si évident, dans certains passage la forme verbale est mieux, dans d'autres la forme nominale, on verra ce que ca donne au final